templates/docs/patterns/grid/index.md

```diff
@@ -25,10 +25,13 @@ Vanilla has a responsive grid with the following columns and gutters:
       <td>1.0rem</td>
     </tr>
     <tr>
-      <td><code>$breakpoint-small</code> - <code>$breakpoint-large</code></td>
-      <td>6</td>
-      <td>2.0rem</td>
-      <td>1.5rem</td>
+      <td rowspan="2"><code>$breakpoint-small</code> - <code>$breakpoint-large</code></td>
+      <td>6 (using <code>.row</code>)</td>
+      <td rowspan="2">2.0rem</td>
+      <td rowspan="2">1.5rem</td>
+    </tr>
+    <tr>
+      <td>4 (using <code>.row--4-cols-medium</code> - see <a href="#4-column-grid-on-medium-screens">docs</a>).</td>
```

Comment:
If that's the new default we should list it first. And maybe add `deprecated` red status label to the cell listing `.row`